.github/CONTRIBUTOR_AGREEMENT.md

```diff
@@ -98,9 +98,9 @@ mark both statements:
 
 | Field                          | Entry                |
 |------------------------------- | -------------------- |
-| Name                           |                      |
+| Name                           | Bharat Raghunathan   |
```

Comment:
Quick note: Looks like you accidentally filled in the *template* instead of copying it over to a new file like `.github/contributors/Bharat123rox.md`. If you could fix that, that'd be great!